<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="7" hostname="934CUFS" failures="1" tests="1" name="Default test" time="0.073" errors="0" timestamp="2023-11-27T23:49:49 CET">
  <testcase classname="com.gestion.ecole.gestionecole.services.CenterServiceTest" name="convertCenterToDtoTest" time="0.016">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;com.gestion.ecole.gestionecole.repositories.EstablishmentRepository.save(Object)&amp;quot; because &amp;quot;this.repo&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.gestion.ecole.gestionecole.repositories.EstablishmentRepository.save(Object)" because "this.repo" is null
at com.gestion.ecole.gestionecole.services.CenterServiceTest.convertCenterToDtoTest(CenterServiceTest.java:21)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- convertCenterToDtoTest -->
  <testcase classname="com.gestion.ecole.gestionecole.services.CenterServiceTest" name="deleteAllTest" time="0.0">
    <ignored/>
  </testcase> <!-- deleteAllTest -->
  <testcase classname="com.gestion.ecole.gestionecole.services.CenterServiceTest" name="findByCriteriaTest" time="0.0">
    <ignored/>
  </testcase> <!-- findByCriteriaTest -->
  <testcase classname="com.gestion.ecole.gestionecole.services.CenterServiceTest" name="deleteTest" time="0.0">
    <ignored/>
  </testcase> <!-- deleteTest -->
  <testcase classname="com.gestion.ecole.gestionecole.services.CenterServiceTest" name="saveOrUpdateTest" time="0.0">
    <ignored/>
  </testcase> <!-- saveOrUpdateTest -->
  <testcase classname="com.gestion.ecole.gestionecole.services.CenterServiceTest" name="convertCycleToDtoTest" time="0.0">
    <ignored/>
  </testcase> <!-- convertCycleToDtoTest -->
  <testcase classname="com.gestion.ecole.gestionecole.services.CenterServiceTest" name="findByIdTest" time="0.0">
    <ignored/>
  </testcase> <!-- findByIdTest -->
  <testcase classname="com.gestion.ecole.gestionecole.services.CenterServiceTest" name="findAllTest" time="0.0">
    <ignored/>
  </testcase> <!-- findAllTest -->
</testsuite> <!-- Default test -->
